scripts/output.js

```diff
@@ -233,7 +233,13 @@ function downloadCSV(csvContent, old) {
 	create_zip(csvContent, old);
 }
 
+/*
+*This function...
```

Comment:
Can you finish the comments here to document what the downloadURI method does? You should mention that if the name or uri passed in are empty, we don't append the image object into the stored image list.